gapps_users_to_contacts_group_copier/gapps_users_to_contacts_group_copier.py

```diff
@@ -4,12 +4,13 @@
 __author__ = "vsin"
 __version__ = "1.0.0"
```

Comment:
Quick googling would agree. PEP8 however only defines __version__ and advices to not invent own names. This __author__ syntax would seem like widespread non-standard use.

I'm more used to defining these with setuptools when creating packages.